app/views/kaminari/_paginator.html.haml

```diff
@@ -1,5 +1,5 @@
 = paginator.render do
-  %ul.pager-items
+  %ul.pager-items{"data-click-events" => "", "data-click-category" => "Register Table", "data-click-action" => "Pagination"}
```

Comment:
if you use `nil` instead of `""` does it generate the same markup?

Reply:
Doesn't render in the DOM if you use nil.

Reply:
how about `"data-click-events" => true`?